frontend/src/testing/pushapi-mock.js

```diff
@@ -1,28 +1,21 @@
 /**
- * @author Stefan Dimitrov (stefan.dimitrov@clouway.com).
- */
+  * @author Stefan Dimitrov (stefan.dimitrov@clouway.com).
+  */
+ angular.module('clouway-push', [])
 
-angular.module('clouway-push')
+   .config(function ($provide, pushApiProvider, $httpBackend) {
```

Comment:
I ran some tests.
In order to access the `$httpBackend` service in the config block:
```js
var $httpBackend = angular.injector(['ngMock']).get('$httpBackend');

$provide.decorator('pushApi', function ($delegate) { ...
```
instead of injecting it in the arguments.